src/api/subscriptions.js

```diff
@@ -86,6 +86,12 @@ export class Subscriptions {
    */
   showOffers(opt_options) {}
 
+  /**
+   * Starts the Offers flow for a subscription update.
+   * @param {!OffersRequest=} opt_options
+   */
+  showUpdateOffers(opt_options) {}
```

Comment:
Need to add documentation on new APIs.

Similar to: https://github.com/subscriptions-project/swg-js/blob/master/docs/subscribe-flow.md

Reply:
We definitely do! I'll start writing a documentation page. But since the new endpoints are still experimental right now, I don't think we need to have the documentation ready before submitting.